lib/common.sh

```diff
@@ -308,7 +308,13 @@ fi
 export KIND_NODE_IMAGE=${KIND_NODE_IMAGE:-"${DOCKER_HUB_PROXY}/kindest/node:${KIND_NODE_IMAGE_VERSION}"}
 
 # Ansible version
-export ANSIBLE_VERSION=${ANSIBLE_VERSION:-"7.0.0"}
+# Older ubuntu version do no support 7.0.0 because of older python versions
+# Ansible 7.0.0 requires python 3.10+
+if [ "${DISTRO}" == "ubuntu22" ]; then
+    export ANSIBLE_VERSION=${ANSIBLE_VERSION:-"7.0.0"}
+else 
+    export ANSIBLE_VERSION=${ANSIBLE_VERSION:-"6.5.0"}
```

Comment:
let's use 6.6.0 ?

Reply:
Ok, seems this is more desirable option as @tuminoid also noted , I will push a patch here.